server/src/dp_queries/dp_libraries/open_dp.py

```diff
@@ -76,17 +76,9 @@ def cost(self, query_json: OpenDPInp) -> tuple[float, float]:
     def query(self, query_json: OpenDPInp) -> str:
         opendp_pipe = reconstruct_measurement_pipeline(query_json.opendp_json)
 
-        match query_json.input_data_type:
-            case OpenDPInputType.DF:
-                input_data = self.private_dataset.get_pandas_df().to_csv(
-                    header=False, index=False
-                )
-            case OpenDPInputType.PATH:
-                input_data = self.private_dataset.get_local_path()
```

Comment:
Do we still need to support `get_local_path()` for private dataset implementations?

Reply:
No, we don't for now. I am wondering if it will be useful in the future but I don't see any use case